localstack/aws/protocol/parser.py

```diff
@@ -423,7 +425,8 @@ def parse(self, request: HttpRequest) -> Tuple[OperationModel, Any]:
         # Extract the URI for the operation and convert it to a regex
         request_uri_regex = self._get_request_uri_regex(operation)
         input_shape: StructureShape = operation.input_shape
-        return operation, self._parse_shape(request, input_shape, instance, request_uri_regex)
+        parsed = self._parse_shape(request, input_shape, instance, request_uri_regex)
+        return operation, parsed if parsed is not None else {}
```

Comment:
nit: i think this statement is really hard to read. initially i thought that it would return either `(operation, parsed)` if parsed is not none, or otherwise `{}`. took me a bit to understand what was going on.
maybe just pulling up
```
if parsed is None:
    parsed = {}
return operation, parsed
```
is less clever, but a bit clearer.